### ISSUE TITLE
V41.3/v41.0Hashrate/TDP drops(locks)after minutes 

### ISSUE THREAD

**Issue by linecrig** [NebuTech/NBMiner#871]
O/S = Win 11
NBMiner V41.3 & V41.0
GPU = Nvidia 3070 ti
GPU Driver = 512.15
MSI Afterburner OC = -425 clock, +200 mclock, PL = 50%
ISP = Connection not stable, DC's once a day~ (LEO Satellite)

Issue: Runs full LHR unlock for 10~ minutes @ 75 MH/s then drops down to 30~ as if LHR unlock failed. TDP will drop from 50% to 38%. Once this happens, I can't get my MH/s back to 75 unless I restart PC - and then it works temporarily again. For the first 24 hours it was working perfect. I only started having trouble May 11th. Nothing changed in between those times, so I checked for update & started using V41.3 which didn't help. 
If there is any further info you wish to know please ask.

**Comment by AjlaaSS** [comment]
about same for me but I will explain exactly what is happening
used 41.0 for days without issue
maybe two days ago switched to 41.3 and it was running same, ok, until today.
today it happened many times, lhr cards suddenly start to work just like lhr without any unlock. half a hashrate, memory controller utilisation 66%, lower power draw and lower memory temperatures. interesting thing is, on that rig all cards are Nvidia but not all are Lhr. I can't know if issue is about some card, or miner stability. It however set all Lhr cards to work without unlock at all.

As today is little more warm day, I suspected ambient temperatures to be issue. So I started lowering memory clocks on all Lhr cards. But I don't belive temperatures was that high at first place at all. Pads replaced, fans 100%, 3070TI was aroun 78-80C, and 3080 cards 82-84C.
Day ago it was all running 2C lower. And I did not need to lower overclocks. So I doubt that 2C difference is issue, as I lowered overclocks much and still run into this. Lowering houndreds of mhz on memory speed would really cope with that 2C difference.
Now testing 41.0 again to see if this is going to happen.

Windows 10
515.15 driver, previous DDU clean.

it however did not require PC reboot. to run miner again was enough to start at full hashrate. but I did try both with restarting PC and without it, same issue.
